flowdb/testdata/bin/9900_ingest_synthetic_data.sh

```diff
@@ -76,3 +76,6 @@ else
     echo "Must set SYNTHETIC_DATA_GENERATOR environment variable to 'sql' or 'python'."
     exit 1
 fi
+if [ "$TEST_QA_CHECK" ]; then
+   pipenv run python ./run_qa_checks.py --dates ${DISASTER_END:-"2015-01-01"} --event-types calls mds sms
```

Comment:
Do we not want to just run it for all the dates in the database really?

Reply:
And all event types

Reply:
Do we have internal sources of truth for those? I'd guess it's `etl.available_dates` for dates, but I'm not sure what it would be for the various flavours of events.

Reply:
`etl.etl_records` or `available_tables`